lib/rules/font-family-generic-fallback/index.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+const declarationValueIndex = require("../../utils/declarationValueIndex");
+const findFontFamily = require("../../utils/findFontFamily");
+const keywordSets = require("../../reference/keywordSets");
+const report = require("../../utils/report");
+const ruleMessages = require("../../utils/ruleMessages");
+const validateOptions = require("../../utils/validateOptions");
+
+const ruleName = "font-family-generic-fallback";
+
+const messages = ruleMessages(ruleName, {
+  generic: () => "Missing generic font family, e.g. serif"
```

Comment:
For `no` rules, [the convention](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#determine-violation-messages) is to use `rejected`. A simple string would suffice too. Can you please use:

```
rejected: "Unexpected missing generic font family"
```

You'll need to update your tests accordingly too.